app-emulation/qemu/qemu-2.11.0.recipe

```diff
@@ -125,6 +138,26 @@ INSTALL()
 {
 	make install $jobArgs
 
+	local APP_SIGNATURE="application/x-vnd.qemu"
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+	local LONG_INFO="$SUMMARY"
+	sed \
+		-e "s|@APP_SIGNATURE@|$APP_SIGNATURE|" \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		-e "s|@LONG_INFO@|$LONG_INFO|" \
+		$portDir/additional-files/qemu.rdef.in > qemu.rdef
+
+	if [ $effectiveTargetArchitecture == x86_64 ]; then
+		addResourcesToBinaries qemu.rdef $binDir/qemu-system-i386
+		addResourcesToBinaries qemu.rdef $binDir/qemu-system-x86_64
+	else
+		addResourcesToBinaries qemu.rdef $binDir/qemu-system-i386
+	fi
+
 	# provide convenience symlink to just "qemu"
 	if [ $effectiveTargetArchitecture == x86_64 ]; then
```

Comment:
What you have works fine, but I'd like to suggest a small variant:
```diff
 	$portDir/additional-files/qemu.rdef.in > qemu.rdef
 
+	addResourcesToBinaries qemu.rdef $binDir/qemu-system-i386
-	if [ $effectiveTargetArchitecture == x86_64 ]; then
+	if [ "$effectiveTargetArchitecture" = x86_64 ]; then
-		addResourcesToBinaries qemu.rdef $binDir/qemu-system-i386
 		addResourcesToBinaries qemu.rdef $binDir/qemu-system-x86_64
-	else
-		addResourcesToBinaries qemu.rdef $binDir/qemu-system-i386
 	fi
 
 	# provide convenience symlink to just "qemu"
-	if [ $effectiveTargetArchitecture == x86_64 ]; then
+	if [ "$effectiveTargetArchitecture" = x86_64 ]; then
 		ln -s qemu-system-x86_64 $binDir/qemu
 	else
 		ln -s qemu-system-i386 $binDir/qemu
 fi
```
(Reminder: `=` is slightly better than `==` because the latter only works with bash, while the former works with both bash and sh.)

Reply:
I think it might be nice to use a [commandBinDir](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#using-the-commandbindir-user-variable).

Reply:
FWIW haikuporter requires bash syntax anyway, it's kind of a moot point to bring this over and over again.

Reply:
The reason why I recommend using sh syntax instead of bash syntax whenever it is possible (and does not cost anything) is that:
1) nobody can tell that haikuporter will always require bash;
2) if we ever choose to switch to another shell for haikuporter, if that other shell is compatible with sh but not with bash, then avoiding bashisms now will save us work and decrease git noise.

I'm not converting any already written recipes. I'm just making suggestions for recipes that are not merged yet.